packages/ag-charts-enterprise/src/features/annotations/annotations.ts

```diff
@@ -291,18 +291,19 @@ export class Annotations extends _ModuleSupport.BaseModuleInstance implements _M
                     tooltipManager.suppressTooltip('annotations');
                 }
 
-                if (index == null || (index != null && index !== previous)) {
-                    ctx.toolbarManager.updateButton('annotations', 'line-menu', { icon: undefined });
-                    ctx.toolbarManager.updateButton('annotations', 'text-menu', { icon: undefined });
-                }
-
-                if (node) {
+                if (node && index != null) {
```

Comment:
I may be missing something, but I don't _think_ you need the `index != null` check here as `node` will be null if `index` is null. From line 279.

Reply:
Yeah I just did that for ts it was complaining that index is undefined